femto/md/simulate.py

```diff
@@ -144,7 +147,11 @@ def simulate_state(
             f"{femto.md.utils.openmm.get_simulation_summary(simulation)}"
         )
         coords = simulation.context.getState(
-            getPositions=True, getVelocities=True, getForces=True, getEnergy=True
+            getPositions=True,
+            getVelocities=True,
+            getForces=True,
+            getEnergy=True,
+            enforcePeriodicBox=enforce_pbc,
```

Comment:
Is there a reason you're not using the `config` object like you are above in hremd.py?

Reply:
hmm yeah this function doesn't have a solo config, but rather a list of configs for each step, so there wasn't really a single good place to put this in a config or without changing the overall func signature that i could see.